### PR TITLE
Move color constants to UIColor extension

### LINKED ISSUE #519: Move GlobalConstants.Colors properties to UIColor extension
See https://github.com/raedahgroup/dcrios/blob/master/Decred%20Wallet/Constants/Constants.swift#L14

### PR DESCRIPTION
**Pull request by quacklabs**
what this PR does:
fixes #519 

- moved colors constant to UIColor extension

### CODE REVIEW

File: Decred Wallet/Extensions/UIColor.swift [hunk 1]
```diff
@@ -10,6 +10,19 @@ import Foundation
 import UIKit
 
 extension UIColor {
+
```
Comment (neddoz) [inline Decred Wallet/Extensions/UIColor.swift]: You can get rid of the struct and just have the custom color called statically off the UIColr. e.g UIColor.orangeColor

Comment (quacklabs) [reply Decred Wallet/Extensions/UIColor.swift]: resolved in 01da249

**Comment by itswisdomagain** [comment]
The PR title should be more descriptive. The issue that was fixed should probably only be mentioned from the issue description, not title.

**Comment by quacklabs** [comment]
> The PR title should be more descriptive. The issue that was fixed should probably only be mentioned from the issue description, not title.

done.

File: Decred Wallet/Constants/Constants.swift [hunk 2]
```diff
@@ -9,21 +9,7 @@
...
-        static let lightGrey = UIColor(red: 236.0/255.0, green: 238.0/255.0, blue: 241.0/255.0, alpha: 1.0)
-        static let lightBlue =  UIColor(red: 206.0/255.0, green: 238.0/255.0, blue: 250.0/255.0, alpha: 1.0)
-        static let separaterGrey = UIColor(red: 224/255, green: 224/255, blue: 224/255, alpha: 1.0)
-        static let greenishGrey = UIColor(hex: "F1F8E9")
-        static let black = UIColor(hex: "000000")
-        static let TextAmount = UIColor(hex: "#091440")
-        static let displayAamount = UIColor(hex: "617386")
-    }
-    
+struct GlobalConstants {
```
Comment (itswisdomagain) [inline Decred Wallet/Constants/Constants.swift]: There seem to be some extra whitespaces added to this line, it should not have been modified at all.

File: Decred Wallet/Extensions/UIColor.swift [hunk 3]
```diff
@@ -10,6 +10,17 @@ import Foundation
 import UIKit
 
 extension UIColor {
+    
+    static let orangeColor = UIColor.init(hex: "fd714a")
+    static let navigationBarColor = UIColor.init(hex: "689F38")
+    static let lightGrey = UIColor(red: 236.0/255.0, green: 238.0/255.0, blue: 241.0/255.0, alpha: 1.0)
+    static let lightBlue =  UIColor(red: 206.0/255.0, green: 238.0/255.0, blue: 250.0/255.0, alpha: 1.0)
+    static let separaterGrey = UIColor(red: 224/255, green: 224/255, blue: 224/255, alpha: 1.0)
```
Comment (itswisdomagain) [inline Decred Wallet/Extensions/UIColor.swift]:
```suggestion
    static let separatorGrey = UIColor(red: 224/255, green: 224/255, blue: 224/255, alpha: 1.0)
```

File: Decred Wallet/Extensions/UIColor.swift [hunk 4]
```diff
@@ -10,6 +10,17 @@ import Foundation
...
+    static let greenishGrey = UIColor.init(hex: "F1F8E9")
+    static let black = UIColor.init(hex: "000000")
+    static let TextAmount = UIColor.init(hex: "#091440")
+    static let displayAamount = UIColor.init(hex: "617386")
```
Comment (itswisdomagain) [inline Decred Wallet/Extensions/UIColor.swift]:
```suggestion
    static let displayAmount = UIColor.init(hex: "617386")
```

File: Decred Wallet/Extensions/UIColor.swift [hunk 5]
```diff
@@ -10,6 +10,17 @@ import Foundation
...
 extension UIColor {
+    
+    static let orangeColor = UIColor.init(hex: "fd714a")
+    static let navigationBarColor = UIColor.init(hex: "689F38")
+    static let lightGrey = UIColor(red: 236.0/255.0, green: 238.0/255.0, blue: 241.0/255.0, alpha: 1.0)
+    static let lightBlue =  UIColor(red: 206.0/255.0, green: 238.0/255.0, blue: 250.0/255.0, alpha: 1.0)
+    static let separaterGrey = UIColor(red: 224/255, green: 224/255, blue: 224/255, alpha: 1.0)
+    static let greenishGrey = UIColor.init(hex: "F1F8E9")
+    static let black = UIColor.init(hex: "000000")
+    static let TextAmount = UIColor.init(hex: "#091440")
```
Comment (itswisdomagain) [inline Decred Wallet/Extensions/UIColor.swift]:
```suggestion
    static let textAmount = UIColor.init(hex: "#091440")
```

File: Decred Wallet/Constants/Constants.swift (hunk 2 above)
Comment (quacklabs) [reply Decred Wallet/Constants/Constants.swift]: resolved in 649c901

File: Decred Wallet/Extensions/UIColor.swift (hunk 3 above)
Comment (quacklabs) [reply Decred Wallet/Extensions/UIColor.swift]: resolved in 649c901

File: Decred Wallet/Extensions/UIColor.swift (hunk 4 above)
Comment (quacklabs) [reply Decred Wallet/Extensions/UIColor.swift]: resolved in 649c901

File: Decred Wallet/Extensions/UIColor.swift (hunk 5 above)
Comment (quacklabs) [reply Decred Wallet/Extensions/UIColor.swift]: resolved in 649c901

File: Decred Wallet/Extensions/UIColor.swift [hunk 6]
```diff
@@ -26,6 +28,14 @@ extension UIColor {
...
         static let darkGray = UIColor.init(hex: "#4e5f70")
         static let transparentThinGray = UIColor.init(hex: "#a4abb1", alpha: 0.3)
+        static let orangeColor = UIColor.init(hex: "fd714a")
+        static let navigationBarColor = UIColor.init(hex: "689F38")
+        static let lightGrey = UIColor(red: 236.0/255.0, green: 238.0/255.0, blue: 241.0/255.0, alpha: 1.0)
+        static let lightBlue =  UIColor(red: 206.0/255.0, green: 238.0/255.0, blue: 250.0/255.0, alpha: 1.0)
+        static let separatorGrey = UIColor(red: 224/255, green: 224/255, blue: 224/255, alpha: 1.0)
+        static let greenishGrey = UIColor.init(hex: "F1F8E9")
+        static let textAmount = UIColor.init(hex: "#091440")
+        static let displayAmount = UIColor.init(hex: "617386")
```
Comment (itswisdomagain) [inline Decred Wallet/Extensions/UIColor.swift]: From the usages in the files affected by this move, I see that only the `textAmount` property is used. The rest can be safely deleted.
We'll also need to rename that property. What color is that really? Is it `grey`? `greyishBrown`? Let's find a better name for it.
Another thing to consider is if any of the other colors previously defined in the `appColors` struct can be used in place of this struct.

Comment (quacklabs) [reply Decred Wallet/Extensions/UIColor.swift]: resolved in f5d6a38

**Pull request merged**